packages/text/src/Text.ts

```diff
@@ -534,7 +548,7 @@ export class Text extends Sprite
      * @param {boolean} [options.texture=true] - Should it destroy the current texture of the sprite as well
      * @param {boolean} [options.baseTexture=true] - Should it destroy the base texture of the sprite as well
      */
-    destroy(options)
+    public destroy(options: IDestroyOptions): void
```

Comment:
this should be `options: IDestroyOptions|boolean`

Reply:
Good catch! Pushed a fix.